firewall/privacy_mapper_test.go

```diff
@@ -302,13 +353,224 @@ func TestPrivacyMapper(t *testing.T) {
 				FailedUpdates: []*lnrpc.FailedUpdate{
 					{
 						Outpoint: &lnrpc.OutPoint{
-							TxidStr:     "097ef666a61919ff3413b3b701eae3a5cbac08f70c0ca567806e1fa6acbfe384",
-							OutputIndex: 2161781494,
+							TxidStr:     obfusTxID0,
+							OutputIndex: uint32(obfusOut0),
+						},
+					},
+				},
+			},
+		},
+		{
+			name:    "WalletBalance Response",
+			uri:     "/lnrpc.Lightning/WalletBalance",
+			msgType: rpcperms.TypeResponse,
+			msg: &lnrpc.WalletBalanceResponse{
+				TotalBalance:              1_000_000,
+				ConfirmedBalance:          1_000_000,
+				UnconfirmedBalance:        1_000_000,
+				LockedBalance:             1_000_000,
+				ReservedBalanceAnchorChan: 1_000_000,
+				AccountBalance: map[string]*lnrpc.WalletAccountBalance{
+					"first": {
+						ConfirmedBalance:   1_000_000,
+						UnconfirmedBalance: 1_000_000,
+					},
+				},
+			},
+
+			expectedReplacement: &lnrpc.WalletBalanceResponse{
+				TotalBalance:              950_100,
+				ConfirmedBalance:          950_100,
+				UnconfirmedBalance:        950_100,
+				LockedBalance:             950_100,
+				ReservedBalanceAnchorChan: 950_100,
+				AccountBalance: map[string]*lnrpc.WalletAccountBalance{
+					"first": {
+						ConfirmedBalance:   950_100,
+						UnconfirmedBalance: 950_100,
+					},
+				},
+			},
```

Comment:
Nit:
Would be great with also providing test coverage with for a the `/lnrpc.Lightning/WalletBalance` but with the  `ClearAmounts` flag set!

Reply:
Added!